### ISSUE TITLE
ENR-EYE NTR: Lower eyelid retraction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#2842]
[moved here] Moved here!//Lower eyelid retraction//term present in HPO (HP:0030802); please add a subclass of eyelid retraction rather than abnormality of the eyelid (PS); parent- Collier sign [level: 8]; term- Lower eyelid retraction [level: 7]

**Comment by LCCarmody** [comment]
Moved term to subclass of 'eyelid retraction'.  Not sure I understand the 'Collier sign' comment.  Should this also be a subclass of lower eyelid retraction?  I originally was going to list Collier sign as only a subclass of upper eyelid retraction. Issues #2840 and #2841.

**Comment by pnrobinson** [comment]
@psergouniotis right now we have 
eyelid retraction
 - lower eyelid retraction
 - upper eyelid retraction
 ----Collier's sign

Is this correct?

**Comment by psergouniotis** [comment]
yes that would be correct: Collier sign [syn Collier's sign] should be a subclass of upper eyelid retraction
